pkgs/applications/networking/sieve-connect/default.nix

```diff
@@ -27,7 +27,7 @@ stdenv.mkDerivation rec {
   installPhase = ''
     mkdir -p $out/bin $out/share/man/man1
     install -m 755 sieve-connect $out/bin
-    gzip -c sieve-connect.1 > $out/share/man/man1/sieve-connect.1.gz
+    install -m 644 sieve-connect.1 $out/share/man/man1
```

Comment:
Why not use installManPage?

Reply:
Replacing one line with three? Sure :-)
https://github.com/NixOS/nixpkgs/pull/133717